File: mydir/src/Entity/Utilisateur.php
<?php

namespace App\Entity;

use App\Repository\UtilisateurRepository;
use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;
use Symfony\Component\Form\FormTypeInterface;

/**
 * @ORM\Entity(repositoryClass=UtilisateurRepository::class)
 */
class Utilisateur
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @ORM\Column(type="string", length=40, nullable=true)
     * @Assert\NotBlank(
     * )
     */
    private $username;

    /**
     * @ORM\Column(type="string", length=128, nullable=true)
    * @Assert\NotBlank(
     * )
     */
    private $password;

    /**
     * @ORM\Column(type="string", length=16, nullable=true)
     * @Assert\NotBlank(
     * )
     */
    private $role;

    /**
     * @ORM\Column(type="string", length=16, nullable=true)
     * @Assert\NotBlank(
     * )
     */
    private $accountStatus;

    /**
     * @ORM\Column(type="integer", nullable=true)
     * @Assert\NotBlank(
     * )
     */
    private $cinPersonne;

    /**
     * @ORM\Column(type="string", length=16, nullable=true)
     * @Assert\NotBlank(
     * )
     */
    private $immatriculeEntreprise;

    /**
     * @ORM\Column(type="string", length=16, nullable=true)
     * @Assert\NotBlank(
     * )
     * @Assert\Length(
     *      min = 2,
     *      max = 50,
     *      minMessage = "Your first name must be at least {{ limit }} characters long",
     *      maxMessage = "Your first name cannot be longer than {{ limit }} characters"
     * )
     */
    private $nomPersonne;

    /**
     * @ORM\Column(type="string", length=16, nullable=true)
     * @Assert\NotBlank(
     * )
     * @Assert\Length(
     *      min = 3,
     *      max = 16,
     *      minMessage = "Your first name must be at least {{ min }} characters long",
     *      maxMessage = "Your first name cannot be longer than {{ max }} characters"
     * )
     */
    private $prenomPersonne;

    /**
     * @ORM\Column(type="string", length=16, nullable=true)
     * @Assert\NotBlank(
     * )@Assert\Length(
     *      min = 3,
     *      max = 16,
     *      minMessage = "Your first name must be at least {{ min }} characters long",
     *      maxMessage = "Your first name cannot be longer than {{ max }} characters"
     * )
     */
    private $libelleEntreprise;

    /**
     * @ORM\Column(type="string", length=10, nullable=true)
     * @Assert\NotBlank(
     * )
     */
    private $sexePersonne;

    /**
     * @ORM\Column(type="date", nullable=true)
     */
    private $dateNaisPersonne;

    /**
     * @ORM\Column(type="string", length=16, nullable=true)
     * @Assert\NotBlank(
     * )
     */
    private $etatSocialePersonne;

    /**
     * @ORM\Column(type="integer", nullable=true)
     * @Assert\NotBlank(
     * )
     */
    private $telephone;

    /**
     * @ORM\Column(type="string", length=128, nullable=true)
     * @Assert\NotBlank(
     * )
     */
    private $adresse;

    /**
     * @ORM\Column(type="string", length=64, nullable=true)
     * @Assert\NotBlank(
     * )
     * @Assert\Email(
     *     message = "The email '{{ $adresse }}' is not a valid email."
     * )
     */
    private $email;

    /**
     * @ORM\Column(type="string", length=64, nullable=true)
     * @Assert\NotBlank(
     * )
     * @Assert\Url(
     *    message = "The url '{{ $email }}' is not a valid url",
     * )
     */
    private $siteweb;

    /**
     * @ORM\Column(type="string", length=64, nullable=true)
     * @Assert\NotBlank(
     * )
     */
    private $domainePersonne;

    /**
     * @ORM\Column(type="string", length=64, nullable=true)
     * @Assert\NotBlank(
     * )
     */
    private $secteurEntreprise;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getUsername(): ?string
    {
        return $this->username;
    }

    public function setUsername(?string $username): self
    {
        $this->username = $username;

        return $this;
    }

    public function getPassword(): ?string
    {
        return $this->password;
    }

    public function setPassword(?string $password): self
    {
        $this->password = $password;

        return $this;
    }

    public function getRole(): ?string
    {
        return $this->role;
    }

    public function setRole(?string $role): self
    {
        $this->role = $role;

        return $this;
    }

    public function getAccountStatus(): ?string
    {
        return $this->accountStatus;
    }

    public function setAccountStatus(?string $accountStatus): self
    {
        $this->accountStatus = $accountStatus;

        return $this;
    }

    public function getCinPersonne(): ?int
    {
        return $this->cinPersonne;
    }

    public function setCinPersonne(?int $cinPersonne): self
    {
        $this->cinPersonne = $cinPersonne;

        return $this;
    }

    public function getImmatriculeEntreprise(): ?string
    {
        return $this->immatriculeEntreprise;
    }

    public function setImmatriculeEntreprise(?string $immatriculeEntreprise): self
    {
        $this->immatriculeEntreprise = $immatriculeEntreprise;

        return $this;
    }

    public function getNomPersonne(): ?string
    {
        return $this->nomPersonne;
    }

    public function setNomPersonne(?string $nomPersonne): self
    {
        $this->nomPersonne = $nomPersonne;

        return $this;
    }

    public function getPrenomPersonne(): ?string
    {
        return $this->prenomPersonne;
    }

    public function setPrenomPersonne(?string $prenomPersonne): self
    {
        $this->prenomPersonne = $prenomPersonne;

        return $this;
    }

    public function getLibelleEntreprise(): ?string
    {
        return $this->libelleEntreprise;
    }

    public function setLibelleEntreprise(?string $libelleEntreprise): self
    {
        $this->libelleEntreprise = $libelleEntreprise;

        return $this;
    }

    public function getSexePersonne(): ?string
    {
        return $this->sexePersonne;
    }

    public function setSexePersonne(?string $sexePersonne): self
    {
        $this->sexePersonne = $sexePersonne;

        return $this;
    }

    public function getDateNaisPersonne(): ?\DateTimeInterface
    {
        return $this->dateNaisPersonne;
    }

    public function setDateNaisPersonne(?\DateTimeInterface $dateNaisPersonne): self
    {
        $this->dateNaisPersonne = $dateNaisPersonne;

        return $this;
    }

    public function getEtatSocialePersonne(): ?string
    {
        return $this->etatSocialePersonne;
    }

    public function setEtatSocialePersonne(?string $etatSocialePersonne): self
    {
        $this->etatSocialePersonne = $etatSocialePersonne;

        return $this;
    }

    public function getTelephone(): ?int
    {
        return $this->telephone;
    }

    public function setTelephone(?int $telephone): self
    {
        $this->telephone = $telephone;

        return $this;
    }

    public function getAdresse(): ?string
    {
        return $this->adresse;
    }

    public function setAdresse(?string $adresse): self
    {
        $this->adresse = $adresse;

        return $this;
    }

    public function getEmail(): ?string
    {
        return $this->email;
    }

    public function setEmail(?string $email): self
    {
        $this->email = $email;

        return $this;
    }

    public function getSiteweb(): ?string
    {
        return $this->siteweb;
    }

    public function setSiteweb(?string $siteweb): self
    {
        $this->siteweb = $siteweb;

        return $this;
    }

    public function getDomainePersonne(): ?string
    {
        return $this->domainePersonne;
    }

    public function setDomainePersonne(?string $domainePersonne): self
    {
        $this->domainePersonne = $domainePersonne;

        return $this;
    }

    public function getSecteurEntreprise(): ?string
    {
        return $this->secteurEntreprise;
    }

    public function setSecteurEntreprise(?string $secteurEntreprise): self
    {
        $this->secteurEntreprise = $secteurEntreprise;

        return $this;
    }
}

File: PiDevKhm/src/entities/utilisateur.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;

import java.sql.Date;

/**
 *
 * @author SPIRIT
 */
public class utilisateur {
    private int id;
    private String password;
    private String account_status;
    private int cin_personne;
    private String immatricule_entreprise;
    private String nom_personne;
    private String prenom_personne;
    private String libelle_entreprise;
    private String sexe_personne;
    private Date date_nais_personne;
    private String etat_sociale_personne;
    private int telephone;
    private String adresse;
    private String email;
    private String siteweb;
    private String domaine_personne;
    private String secteur_entreprise;
    private String role;

    public utilisateur(int id, String password, String account_status, int cin_personne, String immatricule_entreprise, String nom_personne, String prenom_personne, String libelle_entreprise, String sexe_personne, Date date_nais_personne, String etat_sociale_personne, int telephone, String adresse, String email, String siteweb, String domaine_personne, String secteur_entreprise, String role) {
        this.id = id;
        this.password = password;
        this.account_status = account_status;
        this.cin_personne = cin_personne;
        this.immatricule_entreprise = immatricule_entreprise;
        this.nom_personne = nom_personne;
        this.prenom_personne = prenom_personne;
        this.libelle_entreprise = libelle_entreprise;
        this.sexe_personne = sexe_personne;
        this.date_nais_personne = date_nais_personne;
        this.etat_sociale_personne = etat_sociale_personne;
        this.telephone = telephone;
        this.adresse = adresse;
        this.email = email;
        this.siteweb = siteweb;
        this.domaine_personne = domaine_personne;
        this.secteur_entreprise = secteur_entreprise;
        this.role = role;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getAccount_status() {
        return account_status;
    }

    public void setAccount_status(String account_status) {
        this.account_status = account_status;
    }

    public int getCin_personne() {
        return cin_personne;
    }

    public void setCin_personne(int cin_personne) {
        this.cin_personne = cin_personne;
    }

    public String getImmatricule_entreprise() {
        return immatricule_entreprise;
    }

    public void setImmatricule_entreprise(String immatricule_entreprise) {
        this.immatricule_entreprise = immatricule_entreprise;
    }

    public String getNom_personne() {
        return nom_personne;
    }

    public void setNom_personne(String nom_personne) {
        this.nom_personne = nom_personne;
    }

    public String getPrenom_personne() {
        return prenom_personne;
    }

    public void setPrenom_personne(String prenom_personne) {
        this.prenom_personne = prenom_personne;
    }

    public String getLibelle_entreprise() {
        return libelle_entreprise;
    }

    public void setLibelle_entreprise(String libelle_entreprise) {
        this.libelle_entreprise = libelle_entreprise;
    }

    public String getSexe_personne() {
        return sexe_personne;
    }

    public void setSexe_personne(String sexe_personne) {
        this.sexe_personne = sexe_personne;
    }

    public Date getDate_nais_personne() {
        return date_nais_personne;
    }

    public void setDate_nais_personne(Date date_nais_personne) {
        this.date_nais_personne = date_nais_personne;
    }

    public String getEtat_sociale_personne() {
        return etat_sociale_personne;
    }

    public void setEtat_sociale_personne(String etat_sociale_personne) {
        this.etat_sociale_personne = etat_sociale_personne;
    }

    public int getTelephone() {
        return telephone;
    }

    public void setTelephone(int telephone) {
        this.telephone = telephone;
    }

    public String getAdresse() {
        return adresse;
    }

    public void setAdresse(String adresse) {
        this.adresse = adresse;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getSiteweb() {
        return siteweb;
    }

    public void setSiteweb(String siteweb) {
        this.siteweb = siteweb;
    }

    public String getDomaine_personne() {
        return domaine_personne;
    }

    public void setDomaine_personne(String domaine_personne) {
        this.domaine_personne = domaine_personne;
    }

    public String getSecteur_entreprise() {
        return secteur_entreprise;
    }

    public void setSecteur_entreprise(String secteur_entreprise) {
        this.secteur_entreprise = secteur_entreprise;
    }

    public String getRole() {
        return role;
    }

    public void setRole(String role) {
        this.role = role;
    }

    @Override
    public String toString() {
        return "utilisateur{" + "id=" + id + ", password=" + password + ", account_status=" + account_status + ", cin_personne=" + cin_personne + ", immatricule_entreprise=" + immatricule_entreprise + ", nom_personne=" + nom_personne + ", prenom_personne=" + prenom_personne + ", libelle_entreprise=" + libelle_entreprise + ", sexe_personne=" + sexe_personne + ", date_nais_personne=" + date_nais_personne + ", etat_sociale_personne=" + etat_sociale_personne + ", telephone=" + telephone + ", adresse=" + adresse + ", email=" + email + ", siteweb=" + siteweb + ", domaine_personne=" + domaine_personne + ", secteur_entreprise=" + secteur_entreprise + ", role=" + role + '}';
    }

    
}
